docs/install/ubuntu-debian.rst

```diff
@@ -27,7 +27,7 @@ Install from PPA repository
 .. code-block:: bash
 
     sudo apt-get install dirmngr
-    sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu cosmic main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-cosmic.list'
+    sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu bionic main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-cosmic.list'
```

Comment:
```suggestion
    sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu bionic main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri.list'
```

Removing "cosmic", but also not replacing it with another dist name will ensure that re-applying instructions later in time won't duplicate repos.